src/pages/restaurant/components/menu/Menu.js

```diff
@@ -0,0 +1,115 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import saveItem from './helpers/saveItem';
+import getUpdatedQuantityPerDishById from './helpers/getUpdatedQuantityPerDishById';
+import getKeysOfAnObject from './helpers/getKeysOfAnObject';
+
+//Constants
+import { INITIAL_STATE } from './constants/menu.general';
+
+//Components
+import MenuLinks from './components/menuLinks';
+import Dishes from './components/dishes';
+import Cart from './components/cart';
+
+//CSS
+import './menu.css';
+
+class Menu extends React.Component {
+  state = INITIAL_STATE;
+  resetCartItems() {
+    this.setState({
+      quantityPerDishById: {},
```

Comment:
your initial state and this state are same for now, so you can initial state here.
in future if that state changes then we can change into whatever we want

Reply:
> your initial state and this state are same for now, so you can use initial state here. in future if that state changes then we can change into whatever we want

yes for now keep it as what you have